pkg/jx/cmd/common_environments.go

```diff
@@ -324,7 +324,7 @@ func (o *CommonOptions) CreateAddRequirementFn(chartName string, alias string, v
 				err = util.CopyFile(valuesFiles[0], rootValuesFileName)
 				if err != nil {
 					return errors.Wrapf(err, "cannot copy values."+
-						"yaml to chartName directory %s", appDir)
+						"yaml to %s directory %s", chartName, appDir)
```

Comment:
Why do you use string concatenation '+' and string interpolation here? I'd remove the '+'

Reply:
That's probably the stupid string wrapper junk in Goland ;-)